apps/server/src/modules/authentication/testing/jwtpayload.factory.ts

```diff
@@ -0,0 +1,67 @@
+import { BaseFactory } from '@shared/testing';
+import { ObjectId } from 'bson';
+import { JwtPayload } from '../interface/jwt-payload';
+
+class JWTPayload implements JwtPayload {

```

Comment:
it would be better if the class has another name as the interface JwtPayload. It could be confusing somewhere else. Maybe: 
```suggestion
class JWT implements JwtPayload {
```